muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/discussion/dialogs/new-thread.tsx

```diff
@@ -181,7 +181,7 @@ class DicussionNewThread extends SessionStateComponent<DicussionNewThreadProps,
        <div className="env-dialog__row" key="3">     
          <div className="env-dialog__form-element-container">
            <div className="env-dialog__label">{this.props.i18n.text.get('plugin.discussion.createmessage.content')}</div>
-           <CKEditor key="3" width="100%" height="grow" growReference=".env-dialog__body" configuration={ckEditorConfig} extraPlugins={extraPlugins}
+           <CKEditor key="3" width="100%" height="210" growReference=".env-dialog__body" configuration={ckEditorConfig} extraPlugins={extraPlugins}
```

Comment:
Modify the ckeditor.tsx file not to use "grow" anymore since after many changes the functionality is broken anyway.

Strip the grow functionality.

Reply:
Should be stripped as well.